cli/src/bin/phylum.rs

```diff
@@ -99,8 +100,9 @@ async fn handle_commands() -> CommandResult {
         }
     }
 
-    if check_for_updates && update::needs_update(false).await {
-        print_update_message();
+    let prerelease = CURRENT_VERSION.contains('-');
```

Comment:
As far as I know, we have never actually updated the cargo version for an RC release, so `prerelease` will always be `false`.